firebase-sessions/firebase-sessions.gradle.kts

```diff
@@ -42,11 +42,11 @@ android {
 }
 
 dependencies {
+  implementation("com.google.firebase:firebase-common-ktx:20.3.2")
+  implementation("com.google.firebase:firebase-components:17.1.0")
+  implementation("com.google.firebase:firebase-encoders-json:18.0.1")
```

Comment:
Changes look good to me here. But curious why we choose pinned versions over a HEAD dependency! Would be a good learning for all of for future.

Reply:
There was an effort across the whole repo to pin dep versions see [go/firebase-android-cross-deps](http://go/firebase-android-cross-deps)